src/crawlee/_types.py

```diff
@@ -26,6 +26,10 @@
 
 HttpMethod: TypeAlias = Literal['GET', 'HEAD', 'POST', 'PUT', 'DELETE', 'CONNECT', 'OPTIONS', 'TRACE', 'PATCH']
 
+HttpQueryParams: TypeAlias = dict[str, Any]
+
+HttpPayload: TypeAlias = dict[str, Any]
```

Comment:
The `dict` is kinda surprising here - I'd expect `bytes` or something. How did this happen? :smile:

Reply:
The downstream libraries (HTTPX & CurlImpersonate) do the conversions. And both of them accept `RequestData = Mapping[str, Any]` type.

Reply:
Right. I did some digging and this is what I found. `payload` is part of the `Request` schema on the API, and the API client requires it to be a string. It'll get serialized and passed to the API endpoint on platform. Of course, it also needs to be compatible with whatever the HTTP client accepts - that's where the actual processing gets done.

As a side note, I'm not sure what `data` was supposed to be, but it seems it's not needed...

Reply:
payload is bytes type now